website/src/pages/case-studies/nutanix-fixed-log4j-with-sourcegraph.tsx

```diff
@@ -105,17 +105,17 @@ const NutanixCaseStudy: FunctionComponent<PageProps> = props => (
                                 <ChartBarIcon color="#00A1C7" size={40} className="p-1" />
                             </div>
                             <div className="pl-3 col-11 col-lg-9">
-                                <h5 className="font-weight-bold">Results</h5>
+                                <h4 className="font-weight-bold">Results</h4>
                                 <ul className="pl-4">
-                                    <li>
+                                    <li className="py-3 h5 font-weight-normal">
                                         Nutanix was able to see where JMSAppender existed, fix it, and send out a
                                         release in less than 5 minutes.
                                     </li>
-                                    <li>
+                                    <li className="py-3 h5 font-weight-normal">
                                         Nutanix was able to deliver patches to its customers that fully remediated the
                                         Log4j vulnerability in under 4 days.
                                     </li>
-                                    <li>
+                                    <li className="py-3 h5 font-weight-normal">
                                         Nutanix was able to identify every instance of Log4j across its sprawling
                                         codebase with 100% confidence.
                                     </li>
```

Comment:
This feels odd to me, but QA would like Results section slightly bigger font to fill the space more compared to the left. Will see how they feel about this. 🤷‍♀️ 
![Screen Shot 2022-04-11 at 1 06 23 PM](https://user-images.githubusercontent.com/59381432/162812233-4fed3253-c056-4b5f-8b01-5e1aa2be17c4.png)

Reply:
I agree, seems a bit off to me as well! My advice would be adding another piece of content instead of increasing the font size to compensate for the extra white space.

Reply:
Good idea. I floated the suggestion on the QA doc, I'll see how design feels @bretthayes

Reply:
Feedback from design was to leave the original version, we may add a quote under the "Results" section later to fill it out more